frontend/src/container/AlertHistory/Statistics/StatsCard/StatsCard.tsx

```diff
@@ -0,0 +1,94 @@
+import './statsCard.styles.scss';
+
+import { DotChartOutlined } from '@ant-design/icons';
+import useUrlQuery from 'hooks/useUrlQuery';
+import { ArrowDownLeft, ArrowUpRight, Calendar } from 'lucide-react';
+import { calculateChange } from 'utils/calculateChange';
+
+type ChangePercentageProps = {
+	percentage: number;
+	direction: number;
+	duration: string | null;
+};
+function ChangePercentage({
+	percentage,
+	direction,
+	duration,
+}: ChangePercentageProps): JSX.Element {
+	if (!percentage || !duration) {
+		return <div />;
+	}
+	return (
+		<div
+			className={`change-percentage ${
+				direction ? 'change-percentage--success' : 'change-percentage--error'
+			}`}
+		>
+			<div className="change-percentage__icon">
+				{direction ? (
+					<ArrowDownLeft size={14} color="var(--bg-forest-500)" />
+				) : (
+					<ArrowUpRight size={14} color="var(--bg-cherry-500)" />
+				)}
+			</div>
+			<div className="change-percentage__label">
+				{percentage}% vs Last {duration}
```

Comment:
% -> use icon

Reply:
@YounixM , in the designs, we have text % (https://www.figma.com/design/jWPCZEqdeA0yDYvixLQ42k/Alerts?node-id=2819-7734&m=dev)